fe1-web/src/core/network/__tests__/NetworkConnection.test.ts

```diff
@@ -105,28 +105,34 @@ describe('NetworkConnection', () => {
     it('resolves to a connection if it is possible to connect', async () => {
       allowNewConnections();
       const connection = await NetworkConnection.create(mockAddress, jest.fn(), jest.fn());
+      const networkConnection = connection[0];
+      const networkError = connection[1];
 
-      expect(connection).toBeInstanceOf(NetworkConnection);
-      expect(connection.address).toEqual(mockAddress);
-      expect(getWebsocket(connection).readyState).toEqual(OPEN);
+      expect(networkConnection).toBeInstanceOf(NetworkConnection);
+      expect(networkConnection.address).toEqual(mockAddress);
+      expect(getWebsocket(networkConnection).readyState).toEqual(OPEN);
+      expect(networkError).toBeNull();
     });
 
     /**
      * This test might log some errors depending on the timeout for the initial connection
      * and the time configured for the mocked websockets
      */
-    it('rejects if it is not possible to create the connection within the timeout', async () => {
+    it('return networkError if it is not possible to create the connection within the timeout', async () => {
```

Comment:
As @Xelowak mentioned in the other PR, it is usually nice to have the 'it' read like a sentence 😁 (in the other tests as well 😅)

Reply:
Oh okay it makes sense, I will keep this in mind for future tests 👍